src/execute_request.jl

```diff
@@ -58,30 +56,50 @@ const displayqueue = Any[]
 
 # remove x from the display queue
 function undisplay(x)
-    i = findfirst(equalto(x), displayqueue)
-    if i > 0
+    i = findfirst(isequal(x), displayqueue)
```

Comment:
Shouldn't this be `Compat.findfirst` to get the version that returns `nothing` on 0.6?